Common/include/option_structure.hpp

```diff
@@ -1886,6 +1886,17 @@ enum TURBO_MARKER_TYPE{
   OUTFLOW = 2     /*!< \brief flag for outflow marker for compute turboperformance. */
 };
 
+enum TURBO_RAMP_TYPE{
+  GRID = 1,       /*!< \brief flag for rotational/translational ramps */
+  BOUNDARY = 2    /*!< \brief flag for pressure/mass flow ramps*/
+};
+
+enum TURBO_RAMP_COEFF{
```

Comment:
Document that these are used to access the entries in the coefficient ramps and thus are numbered in a particular way.